packages/synapse-react-client/src/lib/containers/row_renderers/Dataset.tsx

```diff
@@ -2,17 +2,20 @@ import * as React from 'react'
 import { DATASET } from '../../utils/SynapseConstants'
 import { calculateFriendlyFileSize } from '../../utils/functions/calculateFriendlyFileSize'
 import * as Utils from './utils'
-import { getValueOrMultiValue } from '../GenericCard'
+import { GenericCardSchema, getValueOrMultiValue } from '../GenericCard'
 import { SelectColumn, ColumnModel } from '../../utils/synapseTypes'
 import { Button } from 'react-bootstrap'
 import { PRODUCTION_ENDPOINT_CONFIG } from '../../utils/functions/getEndpoint'
+import { QueryVisualizationContextType } from '../QueryVisualizationWrapper'
 
 export type DatasetProps = {
   data?: any
   schema?: any
   secondaryLabelLimit?: number
   selectColumns?: SelectColumn[]
   columnModels?: ColumnModel[]
+  genericCardSchema?: GenericCardSchema
+  queryVisualizationContext: QueryVisualizationContextType
```

Comment:
The Card already attempts to pass these into the special row renderers, we just need to define them in the props to gain access to them.